deepchem/models/torch_models/tests/test_highway.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+try:
+    import torch
+    import numpy as np
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+    pass
+from deepchem.models.torch_models.layers import Highway
+
+
+@pytest.mark.torch
+def test_highway_shape_no_params():
```

Comment:
`RuntimeError: The size of tensor a (5) must match the size of tensor b (2) at non-singleton dimension ` error in the tests